packages/template-1c/package.json

```diff
@@ -12,16 +15,17 @@
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
         "react-helmet": "^6.1.0",
-        "react-router-dom": "^5.1.2"
+        "react-router-dom": "^5.1.2",
+        "typescript": "^4.5.5"
     },
     "scripts": {
         "format": "mrt format '\"**/*.{js,jsx}\"'",
-        "lint": "mrt lint '\"**/*.{js,jsx}\"'",
+        "lint": "tsc && mrt lint '\"**/*.{js,jsx}\"'",
```

Comment:
The common pattern to run typechecking only with `tsc` is to run `tsc --noEmit` to prevent it from creating files
https://www.typescriptlang.org/tsconfig/#noEmit
you could set that in `tsconfig.json`, but other tools like babel also use tsconfig.json
As a nice thing you can make it its own script and reference that in other scripts
https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/24b516db3cec94bb40e856a39f696a081b3d90f3/package.json#L26